src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
+    public Random getRandom() {
+        return random;
+    }
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return color.toString();
```

Comment:
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)